src/main/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/config/ResourceServerConfiguration.kt

```diff
@@ -32,18 +34,16 @@ class ResourceServerConfiguration(
           customizer.anyRequestRoleCustomizer.defaultRole
             ?.also { authorize(anyRequest, hasRole(it)) }
             ?: also {
-              // TODO: This has to be a user role - not ideal but will secure with this for now
-              authorize("/report/**", hasRole(dprEndpointApiRole))
-              authorize("/reports/**", hasRole(dprEndpointApiRole))
-              authorize("/definitions/**", hasRole(dprEndpointApiRole))
-              authorize("/statements/**", hasRole(dprEndpointApiRole))
+              authorize("/report/**", hasAnyRole(*dprEndpointApiRoles.toTypedArray()))
```

Comment:
Restrict to all the roles that a user can have - I would have preferred to restrict the endpoints using a system token, but that requires DPR library to check use roles in a different way.  For now I've protected the API endpoints by the same roles as the IRS service.